app/src/main/java/org/p2p/wallet/swap/jupiter/statemanager/price_impact/SwapPriceImpact.kt

```diff
@@ -0,0 +1,23 @@
+package org.p2p.wallet.swap.jupiter.statemanager.price_impact
+
+import java.math.BigDecimal
+import org.p2p.core.utils.isLessThan
+
+enum class SwapPriceImpact {
+
+    NORMAL, YELLOW, RED
+}
+
+private val threePercent
+    get() = BigDecimal.valueOf(0.3)
+
+private val onePercent
+    get() = BigDecimal.valueOf(0.1)
+
+fun BigDecimal.toPriceImpactType(): SwapPriceImpact {
```

Comment:
not obvious, I suggest to move this method to SwapPriceImpact

Reply:
SwapPriceImpact is enum, not support companion object. moved to presenter

Reply:
`SwapPriceImpact.parseValue(impact: BigDecimal)`